src/components/layouts/registerDialog.tsx

```diff
@@ -0,0 +1,127 @@
+import { Field } from "@/components/ui/field";
+import {
+  DialogBody,
+  DialogCloseTrigger,
+  DialogContent,
+  DialogTitle,
+  DialogRoot,
+  DialogHeader,
+} from "@/components/ui/dialog";
+import { Stack, Input, Text } from "@chakra-ui/react";
+import { LuUserPlus } from "react-icons/lu";
+import { Button } from "@/components/ui/button";
+import useMainStore from "@/hooks/main/store";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import useAuthStore from "@/contexts/auth/store";
+import clsx from "clsx";
+import useUserStore from "@/contexts/user/store";
+import { CreateUserDto } from "@/contexts/user/types/create";
+import { CreateUserSchema } from "@/contexts/user/schemas/create";
+import { CreateUserField } from "@/contexts/user/fields/create";
```

Comment:
create an alias import for each contexts similar like this:
https://github.com/PAWII-Portal-Loker/backend/blob/master/tsconfig.json

so it have a better looks:
```js
import { CreateUserDto } from "@user/types/create";
import { CreateUserSchema } from "@user/schemas/create";
import { CreateUserField } from "@user/fields/create";
```